src/locale/misc/smartSearch/en.yml

```diff
@@ -21,6 +22,14 @@ most_active:
     examples:
         one: Add the 100 most active members of the organization before today.
         two: Remove the 5 most active members of the organization at any point in time.
+random:
+    inputString: Randomly {addRemoveSelect} {quantity} in the organization.
```

Comment:
Maybe we need it there for the percentage sentence to make sense?

Reply:
Yes, I had it there exactly for the percentage, saying 'randomly add x% of people' sounds super weird (even though correct it reads more like the entire human population rather than a specific group.' It would read better otherwise without though, maybe I can include it in the 'quantity' select and have it conditionally.

Reply:
But then what do you think about the sentence changing that much on changing the select?

Reply:
Nah lets keep it simple and leave it as is. 😊